src/20_Components/amp-facebook-page.html

```diff
@@ -0,0 +1,75 @@
+<!--
+  ## Introduction
+
+  The amp-facebook-page component allows embedding Facebook pages into AMP files.
```

Comment:
Can you please link to the official docs.

Reply:
Done